auth/server/src/main/java/com/seagates3/dao/ldap/LdapConnectionManager.java

```diff
@@ -69,6 +69,7 @@ public static void initLdap()
             String msg = "Failed to initialise LDAP.\n" + ex.toString();
             if (retryLdapConnection(ex.getResultCode())) {
               LOGGER.error(msg);
+              LOGGER.debug(msg);
```

Comment:
Why we need debug log when error is already here ? 

Are we missing something ? Error log level has more severity then debug, so error logging is enough.